tools/not.cpp

```diff
@@ -56,20 +59,23 @@ int main(int argc, char* const* argv) {
     return 1;
 
   int result;
-#if !defined(TARGET_OS_IPHONE)
+#ifdef _WIN32
   std::stringstream ss;
   ss << argv[0];
   for (int i = 1; i < argc; ++i)
     ss << " " << argv[i];
   std::string cmd = ss.str();
   result = std::system(cmd.c_str());
-#else
+#elif defined(__unix__) || defined(__APPLE__)
   pid_t pid;
   if (posix_spawn(&pid, argv[0], NULL, NULL, argv, NULL))
     return EXIT_FAILURE;
   if (waitpid(pid, &result, WUNTRACED | WCONTINUED) == -1)
     return EXIT_FAILURE;
+#else
+  #error "Unsupported system"
 #endif
+
   int retcode = 0;
   int signal = 0;
```

Comment:
I think this should be:

```
  int result;
#if defined(__unix__) || defined(__APPLE__)
  pid_t pid;
  if (posix_spawn(&pid, argv[0], NULL, NULL, argv, NULL))
    return EXIT_FAILURE;
  if (waitpid(pid, &result, WUNTRACED | WCONTINUED) == -1)
    return EXIT_FAILURE;
#else
  std::stringstream ss;
  ss << argv[0];
  for (int i = 1; i < argc; ++i)
    ss << " " << argv[i];
  std::string cmd = ss.str();
  result = std::system(cmd.c_str());
#endif

  int retcode = 0;
  int signal = 0;
```

As I am not sure that `defined(_WIN32) || defined(__unix__) || defined(__APPLE__)` covers everything that _was_ supported prior to this patch.

Reply:
Nice catch. Thanks for taking a close look.